x/collection/module/module.go

```diff
@@ -0,0 +1,184 @@
+package module
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/gorilla/mux"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	abci "github.com/line/ostracon/abci/types"
+	"github.com/spf13/cobra"
+
+	"github.com/line/lbm-sdk/client"
+	"github.com/line/lbm-sdk/codec"
+	codectypes "github.com/line/lbm-sdk/codec/types"
+	sdk "github.com/line/lbm-sdk/types"
+	"github.com/line/lbm-sdk/types/module"
+	"github.com/line/lbm-sdk/x/collection"
+
+	"github.com/line/lbm-sdk/x/collection/client/cli"
+	"github.com/line/lbm-sdk/x/collection/keeper"
+)
+
+var (
+	_ module.AppModule      = AppModule{}
+	_ module.AppModuleBasic = AppModuleBasic{}
+)
+
+// AppModuleBasic defines the basic application module used by the collection module.
+type AppModuleBasic struct{}
+
+// Name returns the ModuleName
+func (AppModuleBasic) Name() string {
+	return collection.ModuleName
+}
+
+// RegisterLegacyAminoCodec registers the collection types on the LegacyAmino codec
+func (AppModuleBasic) RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {}
+
+// DefaultGenesis returns default genesis state as raw bytes for the collection
+// module.
+func (AppModuleBasic) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
+	return cdc.MustMarshalJSON(collection.DefaultGenesisState())
+}
+
+// ValidateGenesis performs genesis state validation for the collection module.
+func (AppModuleBasic) ValidateGenesis(cdc codec.JSONCodec, config client.TxEncodingConfig, bz json.RawMessage) error {
+	var data collection.GenesisState
+	if err := cdc.UnmarshalJSON(bz, &data); err != nil {
+		return fmt.Errorf("failed to unmarshal %s genesis state: %w", collection.ModuleName, err)
+	}
+
+	return collection.ValidateGenesis(data)
+}
+
+// RegisterRESTRoutes registers all REST query handlers
+func (AppModuleBasic) RegisterRESTRoutes(clientCtx client.Context, r *mux.Router) {}
+
+// RegisterGRPCGatewayRoutes registers the gRPC Gateway routes for the collection module.
+func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *runtime.ServeMux) {
+	if err := collection.RegisterQueryHandlerClient(context.Background(), mux, collection.NewQueryClient(clientCtx)); err != nil {
+		panic(err)
+	}
+}
+
+// GetQueryCmd returns the cli query commands for this module
+func (AppModuleBasic) GetQueryCmd() *cobra.Command {
+	return cli.NewQueryCmd()
+}
+
+// GetTxCmd returns the transaction commands for this module
+func (AppModuleBasic) GetTxCmd() *cobra.Command {
+	return nil
+	// return cli.NewTxCmd()
+}
+
+func (b AppModuleBasic) RegisterInterfaces(registry codectypes.InterfaceRegistry) {
+	collection.RegisterInterfaces(registry)
+}
+
+//____________________________________________________________________________
```

Comment:
These separator lines have been removed from cosmos-sdk.
How about removing this line for unity?